### PR TITLE
VNC

### PR DESCRIPTION
**Pull request by MatthewZMSU**
# Description

Added VNC in Dockerfile to watch the contexts in work.
Fixed slow loading of `yarn install`.
Updated dependencies.
Updated README.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

Tested it with TightVNC.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: helpers/utils.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ async function newContext(browser, options = {}) {
     }
 
     try {
-        const context = await browser.createIncognitoBrowserContext(options);
+        const context = await browser.createBrowserContext(options);
```
Comment (mxsnq) [inline helpers/utils.js:125]: why? this will share cookies between all contexts

Comment (MatthewZMSU) [reply helpers/utils.js:125]: There is no `createIncognitoBrowserContext` method in puppeteer@22.15.0

Comment (MatthewZMSU) [reply helpers/utils.js:125]: https://pptr.dev/api/puppeteer.browser.createbrowsercontext/#:~:text=This%20won%27t%20share%20cookies/cache%20with%20other%20browser%20contexts.

File: .dockerignore [hunk 2]
```diff
@@ -1,3 +1,3 @@
 node_modules
 .gitignore
-.git
```
Comment (Yatskov) [inline .dockerignore:3]: Why this change?

File: helpers/utils.js (hunk 1 above)
Comment (Yatskov) [inline helpers/utils.js:125]: Why not incognito?

**Review by Yatskov — DISMISSED** [review]

**Review by Yatskov — APPROVED** [review]
